lua/wikis/marvelrivals/Infobox/League/Custom.lua

```diff
@@ -0,0 +1,90 @@
+---
+-- @Liquipedia
+-- wiki=marvelrivals
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Link = require('Module:Widget/Basic/Link')
+local String = require('Module:StringUtils')
+
+local Injector = Lua.import('Module:Widget/Injector')
+local League = Lua.import('Module:Infobox/League')
+
+local Widgets = require('Module:Widget/All')
+local Cell = Widgets.Cell
+
+---@class MarvelrivalsLeagueInfobox: InfoboxLeague
+local CustomLeague = Class.new(League)
+local CustomInjector = Class.new(Injector)
+
+local DEFAULT_PLATFORM = 'PC'
+local PLATFORM_ALIAS = {
+	console = 'Console',
+	pc = 'PC',
+}
+
+---@param frame Frame
+---@return Html
+function CustomLeague.run(frame)
+	local league = CustomLeague(frame)
+	league:setWidgetInjector(CustomInjector(league))
+
+	return league:createInfobox()
+end
+
+---@param id string
+---@param widgets Widget[]
+---@return Widget[]
+function CustomInjector:parse(id, widgets)
+	local caller = self.caller
+	local args = caller.args
+
+	if id == 'custom' then
+		Array.appendWith(widgets,
+			Cell{name = 'Platform', content = {caller:_createPlatformCell(args)}}
+		)
+	end
+	return widgets
+end
+
+---@param lpdbData table
+---@param args table
+---@return table
+function CustomLeague:addToLpdb(lpdbData, args)
+	lpdbData.extradata.platform = args.platform
+
+	return lpdbData
+end
+
+---@param platform string?
+---@return string?
+function CustomLeague:_platformLookup(platform)
+	if String.isEmpty(platform) then
+		platform = DEFAULT_PLATFORM
+	end
+	---@cast platform -nil
+
+	return PLATFORM_ALIAS[platform:lower()]
+end
+
+---@param args table
+---@return string?
+function CustomLeague:_createPlatformCell(args)
+	local platform = self:_platformLookup(args.platform)
+
+	if String.isNotEmpty(platform) then
+		return Link{
+			link = ':Category:' .. platform .. ' Tournaments',
+			children = { platform }
+		}
+	else
+		return nil
+	end
```

Comment:
```suggestion
	if String.isEmpty(platform) then
		return nil
	end

	return Link{
		link = ':Category:' .. platform .. ' Tournaments',
		children = { platform }
	}
```

Reply:
The category is not being set, so why are we linking to it?

Reply:
> The category is not being set, so why are we linking to it?

The intention is to apply the category, and link to it (maybe later ill make a better page that shows a list of all of them). Is there a step i missed in setting this up?